ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -101,6 +101,17 @@ interface CustomActions {
     [key: string]: (...args: any[]) => TestControllerPromise
 }
 
+interface TestInfo {
+    name: string;
+    meta: Record<string, string>;
```

Comment:
Create a separate type alias for the `meta` property and use it for `FixtureFn`, `TestFn`, `TestInfo`, and `FixtureInfo` aliases.